CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-authn-token-http ChangeLog
 
+## 1.4.0 - 2021-01-TBD
+
+### Changed
+- Update peerDependencies to include bedrock-account@5 and bedrock@4.
+- Bake clientID into a cookie.
```

Comment:
I believe the cookie handling is a new feature and therefore should be listed under an `### Added` heading.  A minor (feature) release will almost always include an added heading.

Reply:
Updated here c305a77.